test/unit/lib/urlutilTest.js

```diff
@@ -485,4 +485,50 @@ describe('urlutil', function () {
       assert.deepEqual(result, {data: 'iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4//8/w38GIAXDIBKE0DHxgljNBAAO9TXL0Y4OHwAAAABJRU//5ErkJggg==', ext: 'png'})
     })
   })
+  describe('shortenUrlToDomainTLD', function () {
+    it('null scenario', function () {
+      const result = urlUtil.shortenUrlToDomainTLD(null)
+      assert.equal(result, null)
+    })
+
+    it('empty string', function () {
+      const result = urlUtil.shortenUrlToDomainTLD('')
+      assert.equal(result, '')
+    })
+
+    it('normal url', function () {
+      const result = urlUtil.shortenUrlToDomainTLD('https://brave.com')
+      assert.equal(result, 'https://brave.com/')
```

Comment:
@jasonrsadler this test should fail. We should not apply / at the end if / was not there from the gecko

Reply:
other unit tests are failing because of that

Reply:
Agree. But something else is appending the trailing '/' at the end of the url during normal browsing.
Example: using latest release type in brianbondy.com and press enter. The trailing '/' gets appended at the end.

As for the tests, will fix.

Let me know what your opinion is regarding the current navigation behavior.